website/content/en/docs/overview/cheat-sheet.md

```diff
@@ -0,0 +1,86 @@
+| `$ operator-sdk init --plugins=<plugin-name>`|           
+| `$ make docker-build IMG=<some-registry>/<project-name>:<tag>`          | Build the operator image      |
+| `$ make docker-build docker-push IMG=<some-registry>/<project-name>:<tag>`      | Build and push the operator image for your registry  |
+| `$ make install`         | Install the CRDs into the cluster |
+| `$ make uninstall`         | Uninstall the CRDs into the cluster |
+| `$ make run`         | Run your controller locally and outside of the cluster. Note that, this will run in the foreground, so switch to a new terminal if you want to leave it running. |
+| `$ make deploy`         | Deploy your project on the cluster |
+| `$ make undeploy`         | Undeploy your project on the cluster |
+
+## To work with bundles and help you out to integrate your project with OLM
+
+For further information check [Operator SDK Integration with Operator Lifecycle Manager][olm-integration].
+
+| Helper   | Description  |
+|-------|-----------|
+| `$ make bundle`          | Create/update the [bundle][bundle] based on the project manifests in the `bundle/` directory. More info [Create a bundle][creating-a-bundle]      |
+| `$ operator-sdk bundle validate ./bundle`          | To validate your [bundle][bundle] spec definition.      |
+| `$ operator-sdk bundle validate ./bundle --select-optional suite=operatorframework` | Validate your bundle against [OperatorHub.io][operatorhub-io] criteria. For further information use the flag.`--help` |
+| `$ operator-sdk olm install` | To install OLM on your cluster for development purposes |
+| `$ operator-sdk olm uninstall` | To uninstall OLM from your cluster |
+| `$ make bundle-build IMG=<some-registry>/<project-name-bundle>:<tag>` | To build your bundle operator image |
+| `$ make bundle-build bundle-push IMG=<some-registry>/<project-name-bundle>:<tag>` | To build and push your bundle operator image |
+| `$ operator-sdk run bundle <some-registry>/<project-name-bundle>:<tag>.` | To deploy your bundle operator using OLM on your cluster for development purposes |
+
+### Updating bundle channels
+ 
+The following examples let you update the [bundle][bundle] with data-informed. For further information also check [Upgrade your Operator][upgrade-project] and see [Channel Naming][channel-namming-doc].  
+ 
+**NOTE:** Note that it will carry over any customizations you have made and ensure a rolling update to the next version of your Operator. 
+
+```sh
+$ make bundle CHANNELS=fast,preview DEFAULT_CHANNEL=stable VERSION=1.0.0 IMG=<some-registry>/<project-name-bundle>:<tag>
+```
+
+```sh
+$ export CHANNELS=fast,preview 
+$ export DEFAULT_CHANNEL=stable
+$ export VERSION=1.0.0
+$ make bundle
+make bundle CHANNELS=fast,preview DEFAULT_CHANNEL=stable VERSION=1.0.0 IMG=<some-registry>/<project-name-bundle>:<tag>
+```
```

Comment:
Redundant
```suggestion
```

Reply:
The goal here is to show both options/examples of a need that is very common. So, because of this, I'd like to keep them.

Reply:
The second example isn't really useful, it's just telling people that environment variables are consumed by make. Wouldn't it be better to have a note saying just that?

Reply:
Good suggestion.